libraries/chain/steem_evaluator.cpp

```diff
@@ -457,6 +457,12 @@ namespace golos { namespace chain {
                 GOLOS_CHECK_LOGIC(wto_itr == wto_idx.end(),
                     logic_exception::cannot_delete_post_with_worker_techspec,
                     "Cannot delete a post with worker techspec.");
+
+                const auto& wto_result_idx = _db.get_index<worker_techspec_index, by_worker_result>();
```

Comment:
To unify call - add new functions: get_worker_result, find_worker_result.
You will use them in Worker API plugin.